### ISSUE TITLE
new module: AGAT/SPFILTERBYORFSIZE

### ISSUE THREAD

**Issue by GallVp** [nf-core/modules#7187]
### Is there an existing module for this?

- [x] I have searched for the existing module

### Is there an open PR for this?

- [x] I have searched for existing PRs

### Is there an open issue for this?

- [x] I have searched for existing issues

### Are you going to work on this?

- [x] If I'm planning to work on this module, I added myself to the `Assignees` to facilitate tracking who is working on the module

**Comment by luisas** [comment]
Hey @GallVp , are you planning to work on this one? Is there any associated PR? :)

**Comment by GallVp** [comment]
Yes, I created it on my custom modules repo and have not moved it here yet: https://github.com/GallVp/nxf-components/blob/main/modules/gallvp/agat/spfilterbyorfsize/main.nf

You can install it from there or please feel free to move it. If not, I'll move it in a day or two.
